src/salaryAndBenefits/SalaryAndBenefits.tsx

```diff
@@ -1,16 +1,35 @@
+"use client"
+
 import styles from "./salaryAndBenefits.module.css";
 import Text from "src/components/text/Text";
 import {SalaryAndBenefits as SalaryAndBenefitsPayload} from "studio/lib/payloads/salaryAndBenefits";
 import Benefit from './components/benefit/Benefit';
+import SalaryCalculator from "./components/salaryCalculator/SalaryCalculator";
+import { useState } from "react";
+import { calculatePension } from "./utils/calculateSalary";
 
 interface SalaryAndBenefitsProps {
   salaryAndBenefits: SalaryAndBenefitsPayload
 }
 
 const SalaryAndBenefits = ({salaryAndBenefits}: SalaryAndBenefitsProps) => {
+  const [salary, setSalary] = useState<string | null>(null);
```

Comment:
This should also be passed around as a number

Reply:
Hm, so normally I'd like bs logic related to the calc. to be within the component itself, but because we know that the result from the calculator will affect graphs and other visuals within the page comp, I believe it to be best to keep it all on parent level instead of having some here and some there which are also dependant on one another. 

So let's keep `SalaryCalculator` as a pure UI component. 

global states could be a possible solution, but we don't yet know the full scope of the website, so it's nice to keep it as simple as possible for now.